### PR TITLE
Add set test

### PR DESCRIPTION
**Pull request by snewcomer**
https://github.com/validated-changeset/validated-changeset/pull/9#discussion_r365536512

### CODE REVIEW

File: tests/unit/changeset-test.js [hunk 1]
```diff
@@ -685,6 +682,60 @@ module('Unit | Utility | changeset', function(hooks) {
...
+    assert.equal(dummyModel.org.usa.ny, 'nil');
+    assert.equal(dummyModel.org.usa.mn, 'MN');
+
+    c.save();
+
+    assert.equal(c.get('org.usa.ny'), 'nil2');
+    assert.equal(c.get('org.usa.mn'), 'undefined');
+    assert.equal(dummyModel.org.usa.ny, 'nil2');
+    assert.equal(dummyModel.org.usa.mn, 'undefined');
+  });
```
Comment (snewcomer) [inline tests/unit/changeset-test.js]: @josemarluedke Does this test look right to you?

Comment (josemarluedke) [reply tests/unit/changeset-test.js]: Yep. Looks good! Interesting that it is passing. Not sure how to capture that bug in a test then.

File: README.md [hunk 2]
```diff
@@ -110,8 +110,12 @@ Then, in your favorite form library, simply pass in the `changeset` in place of
 ```hbs
 {{! dummy-form/template.hbs}}
 <form>
-  <Input @value={{changeset.firstName}} />
-  <Input @value={{changeset.lastName}} />
+  <input
+    value={{model.firstName}}
+    oninput={{action (changeset-set this.changeset "model.lastName") value="target.value"}}>
```
Comment (snewcomer) [inline README.md]: @josemarluedke Do these README updates look ok to you?  ie. the `model` as the `value` as noted https://github.com/poteto/ember-changeset/issues/398#issuecomment-573366958

Comment (josemarluedke) [reply README.md]: I don't think this change is correct. I believe forms should always reflect the in-flight model. This way, we can revert changes and have forms reflect it without any issues. Also, If you have a form in a tab layout for example, and you hide half of the form then go back to that, the form would show the incorrect data, the user would think they have lost their changes.

Comment (snewcomer) [reply README.md]: reverted.  Thanks a ton for the insight.

**Pull request merged**